### PR TITLE
Doc 164 try it out

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: docs/try-it-out/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Double click a node to explore its settings and how it processes data.
+
+## Step three: Run the workflow
+
+Run the workflow to check it's working, and load in data for the next steps: select **Execute workflow**.
+
+## Step four: Add a node
+
+Add a third node to message each customer and tell them their description. TheCustomer Messenger node allows you to send a message to fake recipients.
```
Comment (lorenanda) [inline docs/try-it-out/quickstart.md:37]:
```suggestion
Add a third node to message each customer and tell them their description. The Customer Messenger node allows you to send a message to fake recipients.
```

File: docs/try-it-out/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Select the **Add node** <span class="inline-image">![Add node icon](/_images/try-it-out/add-node-small.png)</span> connector on the Set node.
+2. Search for **Customer Messenger**. n8n shows a list of nodes that match the search.
+3. Select **Customer Messenger (n8n training)** to add the node to the canvas. n8n opens the node automatically.
+4. You're going to use [expressions](/code-examples/expressions/) to map in the **Customer ID** and create the **Message**:
+    1. Next to **Customer ID**, select **Parameter options** <span class="inline-image">![Parameter options icon](/_images/try-it-out/parameter-options.png)</span> > **Add Expression**. n8n opens the expressions editor for this field.
```
Comment (lorenanda) [inline docs/try-it-out/quickstart.md:43]:
```suggestion
    1. Next to the **Customer ID** field, select **Parameter options** <span class="inline-image">![Parameter options icon](/_images/try-it-out/parameter-options.png)</span> > **Add Expression**. n8n opens the expressions editor for this field.
```

Without mentioning the field, I tend to look for information right next to the text.
Also, the first icon next to the field is a warning, not the gear for parameter options (but below next to the Message field they're in reverse order – confusing).

File: docs/try-it-out/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    2. Select **Current Node** > **Input Data** > **JSON** > **customer_ID**. n8n adds the expression to the **Expression** editor, and displays a sample output.
+    3. Close the expressions editor.
+    4. Next to **Message**, select **Parameter options** <span class="inline-image">![Parameter options icon](/_images/try-it-out/parameter-options.png)</span> > **Add Expression**. n8n opens the expressions editor for this field.
+    5. Copy this expression into the editor:
+        ```
+        Hi {{$json.customer_name}},  Your description is {{$json.customer_description}}
+        ```
+5. Close the expressions editor, then close the **Customer Messenger** node.
```
Comment (lorenanda) [inline docs/try-it-out/quickstart.md:51]:
```suggestion
5. Close the expressions editor, then close the **Customer Messenger** node by clicking outside the node or selecting Back to canvas.
```

Since there's no X icon for closing like in the previous step, I think it would be more clear to say how to close the node. Though my suggestion might be too wordy.

File: docs/try-it-out/longer-introduction.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+6. You can now check that the node is working and returning the expected date: select **Execute node** to run the node manually. n8n tests the data against the condition, and shows which results match true or false in the **OUTPUT** panel.
+
+!!! note "Weeks without large solar flares"
+    In this tutorial, you are working with live date. If you find there are no X class solar flares when you run the workflow, try replacing **X** in **Value 2** with either **A**, **B**, **C**, or **M**. 
+
+## Step six: Output data from your workflow
+
+The last step of the workflow is to send the two reports about solar flares. For this example, you'll send data to [Postbin](https://www.toptal.com/developers/postbin/). Postbin is a service that receives data and displays it on a temporary web page. 
+
+1. On the If node, select the **Add node** <span class="inline-image">![Add node icon](/_images/try-it-out/add-node.png)</span> connector labelled **true**.
```
Comment (lorenanda) [inline docs/try-it-out/longer-introduction.md:104]:
```suggestion
1. On the If node, select the **Add node** <span class="inline-image">![Add node icon](/_images/try-it-out/add-node.png)</span> connector labeled **true**.
```

**Review by lorenanda — APPROVED** [review]
Looks good! I just made a few suggestions for clarifying some instructions, if you think they make sense.

File: docs/try-it-out/quickstart.md (hunk 2 above)
Comment (StarfallProjects) [reply docs/try-it-out/quickstart.md:43]: I'm going to leave this one as is. It's a common convention to just use the UI name, rather than saying "the field" / "the button" etc. constantly (keeps the writing simpler and cleaner)

File: docs/try-it-out/longer-introduction.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+11. The expression is: `{{$json["classType"]}}`. Add a message to it, so that the full expression is:
+    ```js
+    There was a solar flare of class {{$json["classType"]}}
+    ```
+13. Close the expressions editor to return to the node.
+14. Close the Postbin node to return to the canvas.
+15. Add another Postbin node, to handle the **false** output path from the If node:
+    1. Hover over the Postbin node, then select **Duplicate node** <span class="inline-image">![Duplicate node icon](/_images/try-it-out/duplicate-node.png)</span> to duplicate the first Postbin node.
+    2. Drag the **false** connector from the If node to the left side of the new Postbin node.
+14. You can now test the entire workflow. Select **Execute Workflow**. n8n runs the workflow, showing each stage in progress.
```
[the remark this answers is not in the document]
Comment (StarfallProjects) [reply docs/try-it-out/longer-introduction.md:123]: Good idea

**Comment by StarfallProjects** [comment]
This should be ready to go now, as soon as we have a new desktop release.
